### PR TITLE
Lims 5809

### PR DESCRIPTION
**Pull request by miayman**
both params Pass

### CODE REVIEW

File: ui_testing/elements.py [hunk 1]
```diff
@@ -207,6 +207,10 @@
                        'value': '//*[@id="custom-accordion-panel"]/div/a/i'},
         'archive': {'method': 'link_text',
                     'value': 'Archive'},
+        'testunit_menu': {'method': 'css_selector',
+                          'value': '[class=\"dropdown \"]'},
```
Comment (0xIslamTaha) [inline ui_testing/elements.py]:
```suggestion
                          'value': '[class="dropdown"]'},
```

File: ui_testing/pages/testunits_page.py [hunk 2]
```diff
@@ -53,7 +60,7 @@ def restore_selected_test_units(self):
         self.base_selenium.click(element='test_units:restore')
         self.confirm_popup()
 
-    def get_active_test_units(self):
+    def get_active_selected_test_units(self):
```
Comment (0xIslamTaha) [inline ui_testing/pages/testunits_page.py]: I'm wondering did u change the calling of this method in the other test cases?

File: ui_testing/testcases/basic_tests/test04_testunits.py [hunk 3]
```diff
@@ -740,7 +740,8 @@ def test022_create_multi_test_units_with_same_name(self):
                                  logger_msg='save {} qualitative test unit'.format(test_unit_name))
 
         self.test_unit_page.create_quantitative_mibi_testunit(name=test_unit_name, method=new_random_method,
-                                                              upper_limit=1000, material_type=material_type, category=category)
+                                                              upper_limit=1000, material_type=material_type,
+                                                              category=category)
```
Comment (0xIslamTaha) [inline ui_testing/testcases/basic_tests/test04_testunits.py]: reformat

File: ui_testing/testcases/basic_tests/test04_testunits.py [hunk 4]
```diff
@@ -754,7 +755,8 @@ def test022_create_multi_test_units_with_same_name(self):
         self.test_plan.open_edit_page(test_plan)
         self.base_selenium.click('test_plan:next')
         self.base_selenium.click('test_plan:add_test_units')
-        test_units = self.base_selenium.get_drop_down_suggestion_list(element='test_plan:test_units', item_text=test_unit_name)
+        test_units = self.base_selenium.get_drop_down_suggestion_list(element='test_plan:test_units',
```
Comment (0xIslamTaha) [inline ui_testing/testcases/basic_tests/test04_testunits.py]: reformat

File: ui_testing/pages/testunits_page.py (hunk 2 above)
Comment (miayman) [reply ui_testing/pages/testunits_page.py]: no, it is most likely due to have old version while creating branch, I didn't change it.

**Pull request merged**